### ISSUE TITLE
Stopped working on windows 10 after create new project

### ISSUE THREAD

**Issue by ttpro1995** [EasyRPG/Editor-Qt#98]
I download the file here https://ci.easyrpg.org/view/Editor-Qt/job/editor-qt-windows/lastSuccessfulBuild/artifact/bin/easyrpg_editor_qt_windows.zip

How to reproduce:
1. Set RTP to C:\Program Files (x86)\Common Files\Enterbrain\RGSS2\RPGVX
2. New project -> Fill in all field as image 
![New_project](http://i.imgur.com/FkPF9Ms.png ) 
3. Click OK
4. ![crash](http://i.imgur.com/PIUjw17.png ) 

I go to Control Panel\All Control Panel Items\Security and Maintenance\Problem Details and get problem detail 

```
Description
Faulting Application Path:	E:\RPGGameProject\EasyRPG-Editor\EasyRPG-Editor.exe

Problem signature
Problem Event Name:	APPCRASH
Application Name:	EasyRPG-Editor.exe
Application Version:	0.0.0.0
Application Timestamp:	5800abb7
Fault Module Name:	EasyRPG-Editor.exe
Fault Module Version:	0.0.0.0
Fault Module Timestamp:	5800abb7
Exception Code:	c0000005
Exception Offset:	0003d807
OS Version:	10.0.14393.2.0.0.768.100
Locale ID:	1033
Additional Information 1:	2beb
Additional Information 2:	2beba6fb4680d73a8c78ca7c24ccdb46
Additional Information 3:	1055
Additional Information 4:	10551cc64bcfba7d2297b81a9c0092f2

Extra information about the problem
Bucket ID:	f49bb7a045500d196558d550d9e04323 (108476175033)

```

If you need any addition information that can help you fix the problem, don't hesitate to ask me. I get it for you.

**Comment by Ghabry** [comment]
The RTP is wrong, you need a 2k or 2k3 RTP. But that's unrelated to the crash.

Thanks for the report, please note that the editor is far from being finished and not useful for end users.

**Comment by carstene1ns** [comment]
This should have been fixed by #100.
